open_event/templates/admin/model/create_event.html

```diff
@@ -24,18 +24,86 @@
                       {% endcall %}
                   </div>
                   <div class="col-md-6">
-                      <div id="map" style="width:400px; height:300px"></div>
+                   <div id="floating-panel">
+                   <input id="address" type="textbox" value="Sydney, NSW">
+                   <input id="submit" type="button" value="Geocode" onclick="saveAddress()">
+    </div>
+			<div id="map" style="width:400px; height:300px"></div>
                   </div>
               </div>
         </div>
     </div>
     <div class="col-md-2"></div>
     <!-- bring in the google maps library -->
-        <script src="http://www.openlayers.org/api/OpenLayers.js"></script>
-        <script src="{{ url_for('static', filename='admin/js/map.js' )}}"></script>
+     <script>
+      var marker;
+      var init_marker;
+function initMap() {
+  var map = new google.maps.Map($("#map"), {
```

Comment:
Your code will probably work if you change this to `$("#map")[0]`, which will get the DOM object versus the jquery object.